tests/e2e/specs/mouseposition.spec.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="cypress" />
+
+const mousePositionSelector = 'div#mouse-position > select'
+const olSelector = '.ol-viewport'
+const olMousePosition = '.ol-mouse-position'
```

Comment:
For CSS selector within Cypress, please check this link : https://docs.cypress.io/guides/references/best-practices#Selecting-Elements
I would prefer that we use dedicated custom HTML attribute, like `data-cy="mouse-position"` so that it is not linked to something that might change later on when we redo the UI (it's the next step when we will have all features done, we will do a cycle or two of UI/UX refinement)

Reply:
OK, for my code. But how do add `data-cy` to OL bits? You have to rely on something already existing, like `class` or `id`, do you ?